src/components/KeyboardSpacer/BaseKeyboardSpacerPropTypes.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Top Spacing is used when there requirement of additional height to view. */
+    topSpacing: PropTypes.number,
+
+    /** Callback to update the value of keyboard status along with keyboard height + top spacing. */
+    onToggle: PropTypes.func,
+
+    /** Keyboard event as per platform specific and also for different event to show/hide keyboard https://reactnative.dev/docs/keyboard#addlistener */
+    /** Pass event name for keyboardShow since iOS and android both has different event to show keyboard. */
```

Comment:
Can we add a link to the docs with the different names for each platform?

```suggestion
    /** Pass keyboardShow event name as a param, since iOS and android both have different event names show keyboard. */
```

Reply:
hey @luacmartins I have added this link https://reactnative.dev/docs/keyboard#addlistener
here we have one specific note that says about platform-specific event support should I add in hiding event as well 

<img width="865" alt="Screenshot 2022-08-18 at 9 41 56 PM" src="https://user-images.githubusercontent.com/47522946/185443444-02107fb2-e9a9-428b-b5e9-42a424a0e99f.png">